tutorials/tutorial8/tutorial.py

```diff
@@ -130,26 +202,24 @@ def fit_pod(self, x):
         self.pod.fit(x)
 
 
-# We highlight that the POD modes are directly computed by means of the singular value decomposition (computed over the input data), and not trained using the back-propagation approach. Only the weights of the MLP are actually trained during the optimization loop.
-
-# In[34]:
+# We highlight that the POD modes are directly computed by means of the singular value decomposition (computed over the input data), and not trained using the backpropagation approach. Only the weights of the MLP are actually trained during the optimization loop.
 
+# In[9]:
 
-poisson_problem = SnapshotProblem()
 
 pod_nn = PODNN(pod_rank=20, layers=[10, 10, 10], func=torch.nn.Tanh)
-pod_nn.fit_pod(u)
+pod_nn.fit_pod(u_train)
 
 pinn_stokes = SupervisedSolver(
```

Comment:
maybe I am wrong, but this is not pinn right?

Reply:
Nope, it is not a PINN, it is a data-driven reduced order model using POD as reduction and a feed-forward neural network as approximation technique (no physics here). We just define the solver and the trainer to train the neural network in POD-NN. I will rename it, maybe `pinn_stokes` is a bit confusing :)